selfdrive/car/mazda/interface.py

```diff
@@ -18,7 +19,7 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=None, expe
     ret.safetyConfigs = [get_safety_config(car.CarParams.SafetyModel.mazda)]
     ret.radarOffCan = True
 
-    ret.dashcamOnly = candidate not in (CAR.CX5_2022, CAR.CX9_2021)
+    ret.dashcamOnly = Params().get_bool('dp_mazda_dashcam_bypass')
```

Comment:
```python
ret.dashcamOnly = candidate not in (CAR.CX5_2022, CAR.CX9_2021) or Params().get_bool('dp_mazda_dashcam_bypass')
```
otherwise even CX5 and CX9 needs to enable toggle